tests/integration/test_integration.py

```diff
@@ -1835,43 +1842,42 @@ def test_ks_2sample_random(engine, random_normal_table, configuration):
 @pytest.mark.parametrize(
     "configuration",
     [
-        ("value_0_1", "value_0_1"),
-        ("value_0_1", "value_02_1"),
-        ("value_0_1", "value_1_1"),
+        ("value_0_1", "value_0_1", 0.0, 1.0),
```

Comment:
Add a comment mentioning how you got the values you have in these parametrizations. Namely numpy/scipy version, compiler, platform/architecture/ Anything that might be helpful to track down possible issues in the future.

Reply:
Good idea 👍